src/sidebar/store/modules/annotations.js

```diff
@@ -125,17 +125,20 @@ const initialState = {
 /** @typedef {typeof initialState} State */
 
 const reducers = {
-  /** @param {State} state */
+  /**
+   * @param {State} state
+   * @param {{ annotations: Annotation[], currentAnnotationCount: number }} action
```

Comment:
`currentAnnotationCount` looks like a code smell to me, but I'm just documenting/typing the existing structure in this PR.